libsql-sqlite3/src/sqliteInt.h

```diff
@@ -324,15 +324,17 @@
 #  endif
 #endif
 
-/*
-** Enable SQLITE_USE_SEH by default on MSVC builds.  Only omit
-** SEH support if the -DSQLITE_OMIT_SEH option is given.
-*/
-#if defined(_MSC_VER) && !defined(SQLITE_OMIT_SEH)
```

Comment:
Why can't we set `-DSQLITE_OMIT_SEH` in the build.rs?

Reply:
It does not help. I tried that on libsql_js branch